src/shared/get-release-notes.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import got from 'got';
+import { major } from 'semver';
+import { PLUGIN_INFO_GET_TIMEOUT } from '../constants';
+
+export async function getReleaseNotes(base: string, filename: string, version: string): Promise<string> {
```

Comment:
style: we typically use the `=>` syntax

`const getReleaseNotes = async (base: string, filename: string, version: string): Promise<string> => {}`

you'll see the word `function` written out all over the codebase for older stuff, though.